### PR TITLE
Got 151 to 854c

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records.

### CODE REVIEW

File: SchleswigGottorf/GotD854b.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="GotD784b" xml:lang="en" type="mss">
```
Comment (eu-genia) [inline SchleswigGottorf/GotD854b.xml:4]: the ID seems wrong

File: SchleswigGottorf/GotD854a.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="GotD784b" xml:lang="en" type="mss">
```
Comment (eu-genia) [inline SchleswigGottorf/GotD854a.xml:4]: the ID seems wrong

File: SchleswigGottorf/GotD854c.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="GotD784c" xml:lang="en" type="mss">
```
Comment (eu-genia) [inline SchleswigGottorf/GotD854c.xml:4]: the ID seems wrong

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: SchleswigGottorf/GotD854c.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply SchleswigGottorf/GotD854c.xml]: done

**Review by eu-genia — APPROVED** [review]

**Pull request merged**